pygenn/genn_groups.py

```diff
@@ -774,11 +786,15 @@ def load(self):
                         # Copy row from non-padded indices into correct location
                         ind[i:i + r] = self.ind[syn:syn + r]
                         syn += r
+                elif self.connectivity_initialiser is not None:
+                    self.ind = ind
```

Comment:
I think using ``self.ind`` and ``self.row_lengths`` for both data set from ``set_sparse_connections`` and for views of GeNN data is a bit dodgy - maybe add ``self._ind`` and ``self._row_lengths`` for the GeNN views and **always** set them in ``load`` (this also maintains backward compatibility)